src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -68,12 +78,12 @@ public String toString() {
 
     @DataBoundConstructor
     public MablStepBuilder(
-            final Secret restApiKey,
+            final String restApiKeyName,
```

Comment:
One more spot for name -> id. The more I look at it the more I think we should fix it. `apiKeyCredentialId` or something along those lines (not sure `rest` detail is needed in the name.